spec/cloud/cloudCodeModuleFile.js

```diff
@@ -0,0 +1,3 @@
+Parse.Cloud.define('cloudCodeInFile', () => {
```

Comment:
Why do we need to add a 3rd file with the same content but a different file name?

Reply:
The tests fail otherwise

Reply:
how so?

Reply:
`- Unhandled promise rejection: ParseError: 141 Invalid function: "cloudCodeInFile`